File: examples/example_square.cpp

#include <iostream>

#include "ditalini.h"

// An example showing how to use the ditalini library
// Philip Mocz (2024)

int main() {
  std::cout << "The square of 17 is: " << ditalini::Simulation(17).square()
            << std::endl;
  return 0;
}

File: python/ditalini/examples/example_square.py
# This is an example of how to our the ditalini package
# Philip Mocz (2024)
#
# Install with:
#   pip install ditalini

import ditalini


def main():

    sim = ditalini.Simulation(17)
    print("The square of 17 is", sim.square())


if __name__ == '__main__':
    main()
